src/vs/platform/configuration/common/configurationModels.ts

```diff
@@ -102,7 +102,7 @@ export class ConfigurationModel implements IConfigurationModel {
 	}
 
 	freeze(): ConfigurationModel {
-		// {{SQL CARBON EDIT}}
+		// {{SQL CARBON EDIT}} @todo anthonydresser needs investigation; we shouldn't need to do this
```

Comment:
typically it's a good idea to add a date to TODO comments so they can safely be removed if left if still in the code after an extended period.

Reply:
good point, I'll add dates